python/tvm/contrib/rpc/server.py

```diff
@@ -92,8 +91,12 @@ def _accept_conn(listen_sock, tracker_conn, ping_period=2):
         # Report resource to tracker
         if tracker_conn:
             matchkey = base.random_key(rpc_key + ":")
-            base.sendjson(tracker_conn,
-                          [TrackerCode.PUT, rpc_key, (port, matchkey)])
+            if custom_addr is not None:
```

Comment:
Since we will be breaking the RPC protocol anyway in this cycle, maybe just always send the custom_addr